lab-davidw/models/bait.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
So one of your resources must have : 'one of its properties must be of type mongoose.Schema.ObjectId and reference an ObjectId for a resource from lab-13'. Per lab instructions.